docs/cristin.ttl

```diff
@@ -68,31 +88,54 @@
 :varbeidhovedkatkode
     a            rdf:Property ;
     rdfs:comment """
-    The main category of the resource,
-    The range of this property is found here: https://api.cristin.no/v2/results/categories?lang=nb,en
+    The general category of the resource. A collection of "varbeidunderkatkode".
     """ ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result .
 
 :varbeidunderkatkode
     a            rdf:Property ;
-    rdfs:comment "The secondary category of the resource. No specific range available at the moment." ;
+    rdfs:comment """
+        The more specific category of the resource. No specific range available at the moment.
+        Each varbeidunderkatkode belongs to only one varbeidhovedkatkode and therefore the value of this field is enough
+        to deduct the exact type of the publication.
+        The range of this property is found here: https://api.cristin.no/v2/results/categories?lang=nb,en
+        """ ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result .
 
 :eierkode_opprettet
     a            rdf:Property ;
-    rdfs:comment "The institution of the person who created the first version of the registration." ;
+    rdfs:comment """
+        eierkode_opprettet and brukernavn are the primary key for idenitfying which user created the entry. Each user can have
+        many usersnames(brukernavn) but only one eierkode.
```

Comment:
s/usersnames/usernames/

But the explanation depends on your definition of "user" and "username" and their relation to what a "person" is. Combinations of "brukernavn"+"eierkode" are unique and represent a user in Cristin. Each user is (ideally, not 100% sure there aren't orphans) linked to one "person" record, but each person record may have an arbitrary number of users linked to it - even from the same eierkode. (I myself had two actual UiO accounts at some time)